requirements/edx/base.txt

```diff
@@ -173,3 +173,6 @@ jsonfield==1.0.3
 
 # Inlines CSS styles into HTML for email notifications.
 pynliner==0.5.2
+
+# django current site middleware with default site
+edx-django-sites-extensions
```

Comment:
added release version, and my understanding is that github.txt only contains packages that we install directly from github and are not available at pypi. But I can use the github version if it makes more sense.